explorer/src/__tests__/controllers/admin/heads.test.ts

```diff
@@ -18,9 +18,11 @@ let server: http.Server
 let db: Connection
 let rb: RequestBuilder
 
+beforeEach(() => {
```

Comment:
Should this not be part of the `beforeAll`? We're re-assigning the `db` variable but `server` still holds a reference to the first instance of `db`